checks/raw/shell_download_validate.go

```diff
@@ -846,15 +827,16 @@ func collectUnpinnedPakageManagerDownload(startLine, endLine uint, node syntax.N
 					EndOffset: endLine,
 					Snippet:   cmd,
 				},
-				Type: checker.DependencyUseTypeNpmCommand,
+				Pinned: asBoolPointer(false),
```

Comment:
i think the hardcoded false here is a bug? Or rather we dont reward npm commands which do call "ci"

Reply:
Ah yes! It is a bug.